src/components/atoms/Accordion/AccordionButton.tsx

```diff
@@ -1,19 +1,23 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
-const AccordionButton = styled.button`
-  background-color: #3c266e;
-  color: white;
-  border-color: #3c266e;
-  display: flex;
-  flex-direction: row;
-  margin: 1px;
-  width: 720px;
-  font-size: 15px;
-  line-height: 50%;
-  border-radius: 1px;
-  height: 50px;
-  position: relative;
-  text-align: left;
-`;
+const AccordionButton = styled.button(props => {
+  const { color } = props;
```

Comment:
This pattern isn't necessary here since you aren't actually using this prop. The previous one should be fine.